test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
+
+const css = require('@webref/css');
+
+const cssValues = [
+  { type: 'property', prop: 'properties', value: 'value' },
+  { type: 'extended property', prop: 'properties', value: 'newValue' },
+  { type: 'descriptor', prop: 'descriptors', value: 'value' },
+  { type: 'value space', prop: 'valuespaces', value: 'value' }
+];
+
+css.parseAll().then(all => {
+  for (const [shortname, json] of Object.entries(all)) {
+    describe(`The ${shortname} entry in @webidl/css`, () => {
+      for (const { type, prop, value } of cssValues) {
+        if (!json[prop]) {
```

Comment:
A typo in `prop` could lead to no tests running here. Since the JSON files always have all of these properties, it works to just remove this bit, and have more confidence in the tests as a result.

Reply:
I was actually thinking that we might want to drop keys from the CSS extracts when values are empty sets, to avoid useless constructs such as `"descriptors": {}`. However, I see how guaranteeing presence also provides additional confidence that the code generates the right keys in the end. I'll update the test accordingly.